dev-tools/packaging/templates/docker/Dockerfile.elastic-agent.tmpl

```diff
@@ -181,6 +181,17 @@ RUN mkdir /app
 RUN chown {{ .user }} /app
 {{- end }}
 {{- end }}
+
+{{- if .unpack_beats }}
+RUN mkdir -p {{ $beatHome }}/data/{{.BeatName}}-{{ commit_short }}/{{ .beats_install_path }} && \
+    for beatPath in {{ $beatHome }}/data/{{.BeatName}}-{{ commit_short }}/downloads/*.tar.gz; do \
+    tar xf $beatPath -C {{ $beatHome }}/data/{{.BeatName}}-{{ commit_short }}/{{ .beats_install_path }} && \
+    rm $beatPath; \
```

Comment:
We should not remove this $beatPath. It's not going to save any space in the image anyway because it is included in a previous layer.

Reply:
True that, removed